pkgs/by-name/ar/arq/package.nix

```
@@ -0,0 +1,51 @@
+{
+  lib,
+  stdenvNoCC,
+  fetchurl,
+  cpio,
+  xar,
+}:
+
+stdenvNoCC.mkDerivation (finalAttrs: {
+  pname = "arq";
+  version = "7.35.1";
+
+  src = fetchurl {
+    url = "https://www.arqbackup.com/download/arqbackup/Arq${finalAttrs.version}.pkg";
+    hash = "sha256-xkrWH2r3DaxsBBdyu0Wj/qzjJaa9DTZCzEaB/nb2WyY=";
+  };
+
+  nativeBuildInputs = [
+    cpio
+    xar
+  ];
+
+  unpackPhase = ''
+    runHook preUnpack
+
+    xar -xf $src
+    zcat client.pkg/Payload | cpio -i
+
+    runHook postUnpack
+  '';
+
+  installPhase = ''
+    mkdir -p $out
+    cp -R Applications $out
+  '';
+
+  dontBuild = true;
+
+  # Arq is notarized
+  dontFixup = true;
+
+  meta = {
+    changelog = "https://www.arqbackup.com/download/arqbackup/arq7_release_notes.html";
+    description = "Arq® automatically backs up all your Macs and Windows PCs.";
```

```suggestion
    description = "Arq® automatically backs up all your Macs and Windows PCs";
```
https://github.com/NixOS/nixpkgs/tree/master/pkgs#meta-attributes

---

Fixed 👍